### DISCUSSION TITLE
Latest Distribution in the Web sidebar File count and size are not displayed?

### DISCUSSION THREAD

**Discussion by tagerwang** [verdaccio/verdaccio#2974] [Ideas]
### Discussed in https://github.com/verdaccio/verdaccio/discussions/2922

<div type='discussions-op-text'>

<sup>Originally posted by **tagerwang** January 19, 2022</sup>
os: MacOS Big Sur / Apple M1
platform: npm
try: Verdaccio tried using V5 and V6, node environments from V12 to V16, The result is the same!    

**This is shown in a screenshot of the official documentation：https://verdaccio.org/zh-CN/docs/webui: **
![image](https://user-images.githubusercontent.com/27122038/150065668-0b97bb91-292b-47b6-b368-5b5eb0d91675.png)  

<br/>
**Not shown on the local server: **

![image](https://user-images.githubusercontent.com/27122038/150066112-47465e24-6c19-45fc-a468-757683360d91.png)

It doesn't exist in Verdaccio Storage either!  why?
![image](https://user-images.githubusercontent.com/27122038/150073478-d5ef8dca-38ff-466f-9899-0b362604a526.png)
</div>

**Comment by juanpicado** [comment]
Hi @tagerwang , I think currently not implemented, this information is being send by the package manager as far I know but is not being recorded if is a private  package. 

I also would like to do have it in some way, but I don't know if yarn and pnpm also send this information. Anyhow that's the update of this.

**Reply by guohenghai** [reply]
I have the same problems, in my local:  File count and size are not displayed too , I Googled and Search a lot, no solution at all ，does anyone who can solve this problem?

**Reply by juanpicado** [reply]
Verdaccio at this point does not record the file size if you publish locally, this might change in the future but that's how it works now. Only display the file size whenever is available from upstream registries.

**Comment by mbtools** [comment]
https://github.com/verdaccio/verdaccio/pull/4600
